### DISCUSSION TITLE
v2022-rc1

### DISCUSSION THREAD

**Discussion by image-et-son** [image-et-son/p600fw#179] [Show and tell]
This development stream is based on Version 2.1 RC3 (last commit from GliGli September 6th 2015).

The version rc1 v2022 (release candidate 1) is feature complete and beta testing is concluded.

For more information and contact see https://prophet600revisited.blogspot.com/

v2022 (RC1) contains some corrections based on feedback from beta 2022 (2):
- Typos and updates of the User manual (thanks to all proof readers!)
- Major fix: compared to prior GliGli firmware version LFO and envelope pitch shifts are now harmonic (#70)
- Fix so that all notes off MIDI CC now also applies to notes played via MIDI (#150)
- Fix so that MIDI CC settings (master vol, master tune, etc.) are accepted in any mode (#175)
- Fix to protect stored patch from Z80 SysEx, ie. requireing patch management mode (#178)
- Fix to use the full pot range for menu parameter external voltage (#177)

v2022 (RC1) contains selected new features (very selective in order not to cause regression problems):
- Vibrato target VCO A or B only (#161)
- Additional LFO re-triggering option (new key) (#171)

In addition, the release candidate includes a MIDI specification documentation.

License
=======

Everything is under GPL v3 license, except files that have their own license in the header
.

<hr /><em>This discussion was created from the release <a href='https://github.com/image-et-son/p600fw/releases/tag/v2022-rc1'>v2022-rc1</a>.</em>

**Comment by mdee7777** [comment]
Thank you for doing this. About to download the new firware and give it a spin.
